rescript-relay-documentation/docs/relay-resolvers.md

```diff
@@ -0,0 +1,170 @@
+---
+  id: relay-resolvers
+  title: Relay Resolvers
+  sidebar_label: Relay Resolvers
+---
+
+#### Recommended background reading
+- [Relay Resolvers](https://relay.dev/docs/guides/relay-resolvers/)
+
+
+## Enabling Experimental Resolvers
+Relay Resolvers are still considered experimental. To use them you must ensure that the `ENABLE_RELAY_RESOLVERS`` runtime feature flag is enabled, and that the `enable_relay_resolver_transform`` feature flag is enabled in your project’s Relay config file.
+
+```js
+// relay.config.js
+module.exports = {
+//...
+  featureFlags: {
+    enable_relay_resolver_transform: true,
+  },
+
+//...
+};
+```
+
+```js
+//SomeComponent.res
+RescriptRelay.relayFeatureFlags.enableRelayResolvers = true
+
+@react.component
+let make = () => {
+  //
+}
+```
+
+
+## Basic Example
+```js
+// UserGreetingResolver
+/**
+ * @RelayResolver
+ *
+ * @onType User
+ * @fieldName greeting
+ * @rootFragment GreetingResolver
```

Comment:
I think there's a more compact notation now that we could perhaps also show. Can't find a link to it now though.